spec/cc/yaml/nodes/engine_spec.rb

```diff
@@ -54,4 +54,31 @@
     check = checks["Style/StringLiteral"]
     check.enabled?.must_equal false
   end
+
+  specify 'assignment of raw value' do
+    config = CC::Yaml.parse! <<-YAML
+engines:
+  rubocop:
+    enabled: true
+    YAML
+
+    config.engines["rubocop"].enabled = false
+    config.engines["rubocop"].enabled?.must_equal false
+  end
```

Comment:
I like this test but I think it's testing something that's a functionality in `mapping`. (So applies to `EngineList` and `LanguageList` as well). What would you think about having the same test but in a different spec - e.g. `mapping_spec.rb`?